recipes/Gdal_CSharp/activate.sh

```diff
@@ -0,0 +1,35 @@
+checkPath () {
+        case ":${!PATH_TYPE}:" in
+                *":$1:"*) return 1
+                        ;;
+        esac
+        return 0;
+}
+
+# Prepend to $PATH
+prependToPath () {
+        for a; do
+                checkPath $a
+                if [ $? -eq 0 ]; then
+                        export ${PATH_TYPE}=$a:${!PATH_TYPE}
+                fi
+        done
+}
+
+# Append to $PATH
+appendToPath () {
+        for a; do
+                checkPath $a
+                if [ $? -eq 0 ]; then
+                        export ${PATH_TYPE}=${!PATH_TYPE}:$a
+                fi
+        done
+}
+
+export PATH_TYPE=MONO_PATH
+if [ -z ${MONO_PATH} ]; then
+    export MONO_PATH=$CONDA_PREFIX/lib
+else
+    export CS_OLD_MONO_PATH=$MONO_PATH
+    appendToPath $CONDA_PREFIX/lib
```

Comment:
Adding CONDA_PREFIX/lib to the global path is not great. Why is this not adjusting MONO_PATH?

Reply:
I noticed this as well but its setting the variable specified in `$PATH_TYPE` (i.e. `MONO_PATH`)

Reply:
Ahhhh thanks!

Reply:
It was changing two types of path originally - thus the abstraction. When I removed the library path I did not remove the $PATH_TYPE abstraction.